File: challenges/4-fizz-buzz/python/lbarcat/Desafio4.py
#Desafio4
número = int(input("Insira um número"))

for x in range(1, número + 1):
  if x % 3 == 0 and x % 5 == 0:
    print("FizzBuzz")
  elif x % 3 == 0:
    print("Fizz")
  elif x % 5 == 0:
    print("Buzz")
  else:
    print(x)

File: challenges/4-fizz-buzz/php/thalesmengue/desafio-4.php
<?php

echo "Insira um valor: ";
$valor = fgets(STDIN);

if (!is_numeric($valor) || $valor == 0) {
    echo "Por favor, insira um número válido";
    exit;
}

for ($i = 1; $i <= $valor; $i++) {
    if (($i % 5 == 0) && ($i % 3 == 0)) {
        echo "FizzBuzz \n";
    } else if ($i % 3 == 0) {
        echo "Fizz \n";
    } else if ($i % 5 == 0) {
        echo "Buzz \n";
    } else {
        echo "$i \n";
    }
}
